cdap-app-fabric/pom.xml

```diff
@@ -142,6 +142,18 @@
     </dependency>
     <dependency>
       <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-api-spark</artifactId>
```

Comment:
I realize now I needed this in test scope because I added a Spark program to the test app. Does it harm to have this dependency?